public/js/fx-bento.js

```diff
@@ -1,303 +1,302 @@
+            return ga("send", "event", "bento", eventAction, eventLabel);
+        }
     }
-  }
-
-  handleEvent(event) {
-    const closeBento = () => {
-      this.handleBentoFocusTrap();
-      window.removeEventListener("resize", this.handleBentoHeight);
-      window.removeEventListener("click", this);
-      document.removeEventListener("keydown", this);
-      this.metricsSendEvent("bento-closed", this._currentSite);
-      this.classList.remove("fx-bento-open");
-      this._bentoWrapper.classList.add("fx-bento-fade-out");
-      setTimeout(() => {
-        this._bentoWrapper.classList.remove("fx-bento-fade-out");
-        this._bentoButton.blur();
-        this.classList = [];
-      }, 500);
-      return;
-    };
-
-    const keydownEvent = (event.type === "keydown");
-    const eventTarget = event.target;
-
-    if (
-      // ignore mouse clicks inside the bento
-      (!keydownEvent && ["fx-bento-content active", "fx-bento-headline", "fx-bento-logo", "fx-bento-headline-logo-wrapper"].includes(eventTarget.className)) ||
+
+    handleEvent(event) {
+        const closeBento = () => {
+            this.handleBentoFocusTrap();
+            window.removeEventListener("resize", this.handleBentoHeight);
+            window.removeEventListener("click", this);
+            document.removeEventListener("keydown", this);
+            this.metricsSendEvent("bento-closed", this._currentSite);
+            this.classList.remove("fx-bento-open");
+            this._bentoWrapper.classList.add("fx-bento-fade-out");
+            setTimeout(() => {
+                this._bentoWrapper.classList.remove("fx-bento-fade-out");
+                this._bentoButton.blur();
+                this.classList = [];
+            }, 500);
+            return;
+        };
+
+        const keydownEvent = (event.type === "keydown");
+        const eventTarget = event.target;
+
+        if (
+        // ignore mouse clicks inside the bento
+            (!keydownEvent && ["fx-bento-content active", "fx-bento-headline", "fx-bento-logo", "fx-bento-headline-logo-wrapper"].includes(eventTarget.className)) ||
       // ignore and don't prevent default behavior on key clicks other than Escape, Down Arrow, and Up Arrow
       (keydownEvent && ![27, 40, 38].includes(event.keyCode))
-      ) {
-      return;
-    }
+        ) {
+            return;
+        }
 
-    const hasParent = (el, selector) => {
-      while (el.parentElement) {
-        el = el.parentElement;
-        if (el.tagName === selector)
-          return el;
-      }
-      return null;
-    };
-
-    // close Bento on mouse clicks outside the Bento menu
-    if (hasParent(event.target, "FIREFOX-APPS") === null) {
-      this._active = !this._active;
-      return closeBento();
-    }
+        const hasParent = (el, selector) => {
+            while (el.parentElement) {
+                el = el.parentElement;
+                if (el.tagName === selector)
+                {return el;}
```

Comment:
nit: I had no idea why this line needed to be wrapped in braces, but it seems to be for the `if` block.